javascripts/govuk/analytics/tracker.js

```diff
@@ -0,0 +1,81 @@
+(function() {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Tracker = function(config) {
+    var setDimension = this.setDimension.bind(this);
```

Comment:
In static `.bind` is using a polyfill for older browsers, this might not work unless the polyfill is present.